src/server/common/components/button/template.njk

```diff
@@ -1,9 +1,10 @@
 {% from "loader/macro.njk" import appLoader %}
 
 <span class="app-button__container">
-  <button class="govuk-button app-button"
+  <button class="govuk-button app-button{% if params.classes %} {{ params.classes }}{% endif %}"
```

Comment:
so this is the button used when you need a loader. Otherwise you just use the gov component button. No need to add classes here

Reply:
done